app/models/user.rb

```diff
@@ -1,7 +1,8 @@
 class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
   attr_accessor :remember_token, :activation_token, :reset_token
 
-  scope :latest_user, ->{order(created_at: :asc)}
+  scope :latest_user, ->{order created_at: :asc}
```

Comment:
```suggestion
  scope :latest_user, ->{order :created_at}
```
asc là mặc định em nhé, nên mình có thể lượt bỏ